apps/base/kustomize.yaml

```diff
@@ -41,4 +41,5 @@ spec:
       CLUSTER_PATH: "${ENVIRONMENT}/${CLUSTER}"
       CLUSTER_FULL_NAME: "${ENVIRONMENT}-${CLUSTER}"
       ISSUER_URL: "${ISSUER_URL:-not-set-yet}"
+      WI_ENVIRONMENT: "${WI_ENVIRONMENT:-$ENVIRONMENT}"
```

Comment:
This should default to `ENVIRONMENT` if `WI_ENVIRONMENT` is not set, which lets us override how workload identity is working without breaking any folder structure being used